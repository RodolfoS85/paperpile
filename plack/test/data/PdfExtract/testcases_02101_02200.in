---
file: ../../../PDFREPOSITORY/002101.pdf
---
file: ../../../PDFREPOSITORY/002102.pdf
---
file: ../../../PDFREPOSITORY/002103.pdf
---
file: ../../../PDFREPOSITORY/002104.pdf
---
file: ../../../PDFREPOSITORY/002105.pdf
---
file: ../../../PDFREPOSITORY/002106.pdf
---
file: ../../../PDFREPOSITORY/002107.pdf
---
file: ../../../PDFREPOSITORY/002108.pdf
---
file: ../../../PDFREPOSITORY/002109.pdf
---
file: ../../../PDFREPOSITORY/002110.pdf
---
file: ../../../PDFREPOSITORY/002111.pdf
---
file: ../../../PDFREPOSITORY/002112.pdf
---
file: ../../../PDFREPOSITORY/002113.pdf
---
file: ../../../PDFREPOSITORY/002114.pdf
---
file: ../../../PDFREPOSITORY/002115.pdf
---
file: ../../../PDFREPOSITORY/002116.pdf
---
file: ../../../PDFREPOSITORY/002117.pdf
---
file: ../../../PDFREPOSITORY/002118.pdf
---
file: ../../../PDFREPOSITORY/002119.pdf
---
file: ../../../PDFREPOSITORY/002120.pdf
---
file: ../../../PDFREPOSITORY/002121.pdf
---
file: ../../../PDFREPOSITORY/002122.pdf
---
file: ../../../PDFREPOSITORY/002123.pdf
---
file: ../../../PDFREPOSITORY/002124.pdf
---
file: ../../../PDFREPOSITORY/002125.pdf
---
file: ../../../PDFREPOSITORY/002126.pdf
---
file: ../../../PDFREPOSITORY/002127.pdf
---
file: ../../../PDFREPOSITORY/002128.pdf
---
file: ../../../PDFREPOSITORY/002129.pdf
---
file: ../../../PDFREPOSITORY/002130.pdf
---
file: ../../../PDFREPOSITORY/002131.pdf
---
file: ../../../PDFREPOSITORY/002132.pdf
---
file: ../../../PDFREPOSITORY/002133.pdf
---
file: ../../../PDFREPOSITORY/002134.pdf
---
file: ../../../PDFREPOSITORY/002135.pdf
---
file: ../../../PDFREPOSITORY/002136.pdf
---
file: ../../../PDFREPOSITORY/002137.pdf
---
file: ../../../PDFREPOSITORY/002138.pdf
---
file: ../../../PDFREPOSITORY/002139.pdf
---
file: ../../../PDFREPOSITORY/002140.pdf
---
file: ../../../PDFREPOSITORY/002141.pdf
---
file: ../../../PDFREPOSITORY/002142.pdf
---
file: ../../../PDFREPOSITORY/002143.pdf
---
file: ../../../PDFREPOSITORY/002144.pdf
---
file: ../../../PDFREPOSITORY/002145.pdf
---
file: ../../../PDFREPOSITORY/002146.pdf
---
file: ../../../PDFREPOSITORY/002147.pdf
---
file: ../../../PDFREPOSITORY/002148.pdf
---
file: ../../../PDFREPOSITORY/002149.pdf
---
file: ../../../PDFREPOSITORY/002150.pdf
---
file: ../../../PDFREPOSITORY/002151.pdf
---
file: ../../../PDFREPOSITORY/002152.pdf
---
file: ../../../PDFREPOSITORY/002153.pdf
---
file: ../../../PDFREPOSITORY/002154.pdf
---
file: ../../../PDFREPOSITORY/002155.pdf
---
file: ../../../PDFREPOSITORY/002156.pdf
---
file: ../../../PDFREPOSITORY/002157.pdf
---
file: ../../../PDFREPOSITORY/002158.pdf
---
file: ../../../PDFREPOSITORY/002159.pdf
---
file: ../../../PDFREPOSITORY/002160.pdf
---
file: ../../../PDFREPOSITORY/002161.pdf
---
file: ../../../PDFREPOSITORY/002162.pdf
---
file: ../../../PDFREPOSITORY/002163.pdf
---
file: ../../../PDFREPOSITORY/002164.pdf
---
file: ../../../PDFREPOSITORY/002165.pdf
---
file: ../../../PDFREPOSITORY/002166.pdf
---
file: ../../../PDFREPOSITORY/002167.pdf
---
file: ../../../PDFREPOSITORY/002168.pdf
---
file: ../../../PDFREPOSITORY/002169.pdf
