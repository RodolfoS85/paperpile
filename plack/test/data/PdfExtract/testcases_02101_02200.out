---
authors: 'Qi, Yan and Ye, Ping and Bader, Joel S'
doi: 10.1186/1471-2105-6-288
title: 'Genetic interaction motif finding by expectation maximization - a novel statistical model for inferring gene modules from synthetic lethality'
---
authors: 'Roca, Xavier and Sachidanandam, Ravi and Krainer, Adrian R.'
doi: 10.1261/rna.2040605
title: Determinants of the inherent strength of human 5 splice sites
---
authors: 'Rodda, David J. and Chew, Joon-Lin and Lim, Leng-Hiong and Loh, Yuin-Han and Wang, Bei and Ng, Huck-Hui and Robson, Paul'
doi: 10.1074/jbc.M502573200
title: Transcriptional Regulation of Nanog by OCT4 and SOX2
---
authors: 'Rosenstiel, Philip and Huse, Klaus and Franke, Andre and Hampe, Jochen and Reichwald, Kathrin and Platzer, Cornelia and Roberts, Roland G and Mathew, Christopher G and Platzer, Matthias and Schreiber, Stefan'
doi: 10.1186/1471-2164-8-472
title: Functional characterization of two novel 5' untranslated exons reveals a complex regulation of NOD2 protein expression
---
authors: 'Rother, M. and Resch, A. and Gardner, W. L. and Whitman, W. B. and Bock, A.'
title: 'Heterologous expression of archaeal selenoprotein genes directed by the SECIS element located in the 3, non-translated region'
---
authors: 'Shi, Huijing and Moore, Peter B.'
title: 'The crystal structure of yeast phenylalanine tRNA at 1.93 Å resolution: A classic structure revisited'
---
authors: 'Sloane, Neil J. A.'
title: Kepler's conjecture confirmed
---
authors: 'Sorek, Rotem and Kunin, Victor and Hugenholtz, Philip'
title: 'CRISPR - a widespread system that provides acquired resistance against phages in bacteria and archaea'
---
authors: 'Stark, Alexander and Brennecke, Julius and Bushati, Natascha and Russell, Robert B. and Cohen, Stephen M.'
doi: 10.1016/j.cell.2005.11.023
title: Animal MicroRNAs Confer Robustness to Gene Expression and Have a Significant Impact on 3 UTR Evolution
---
authors: 'Tenson, Tanel and Hauryliuk, Vasili'
doi: 10.1111/j.1365-2958.2009.06741.x
title: Does the ribosome have initiation and elongation modes of translation?
---
authors: 'Updegrove, Taylor and Wilf, Nabil and Sun, Xueguang and Wartell, Roger M.'
doi: 10.1021/bi800479p
title: "Effect of Hfq on RprA-rpoS mRNA Pairing: Hfq-RNA Binding and the Influence of the 5' rpoS mRNA Leader Region"
---
authors: 'Valentonyte, Ruta and Hampe, Jochen and Huse, Klaus and Rosenstiel, Philip and Albrecht, Mario and Stenzel, Annette and Nagy, Marion and Gaede, Karoline I and Franke, Andre and Haesler, Robert and Koch, Andreas and Lengauer, Thomas and Seegert, Dirk and Reiling, Norbert and Ehlers, Stefan and Schwinger, Eberhard and Platzer, Matthias and Krawczak, Michael and Muller-Quernheim, Joachim and Schurmann, Manfred and Schreiber, Stefan'
doi: 10.1038/ng1519
title: Sarcoidosis is associated with a truncating splice site mutation in BTNL2
---
authors: 'Vandepoele, Klaas and De Vos, Wouter and Taylor, John S. and Meyer, Axel and Van de Peer, Yves'
doi: 10.1073/pnas.0307968100
title: 'Major events in the genome evolution of vertebrates: Paranome age and size differ considerably between ray-finned fishes and land vertebrates'
---
authors: 'Vogel, Jörg'
doi: 10.1111/j.1365-2958.2009.06806.x
title: An RNA trap helps bacteria get the most out of chitosugars
---
authors: 'Wang, Fugao and Landau, D. P.'
doi: 10.1103/PhysRevE.64.0461XX
title: 'Determining the density of states for classical statistical models: A random walk algorithm to produce a flat histogram'
---
authors: 'Wang, Changyu and {v SHU-YUN LE} and Ali, Naushad and Siddiqup, Aleem'
title: An RNA pseudoknot is an essential structural element of the internal ribosome entry site located within the hepatitis C virus 5' noncoding region
---
authors: 'Xie, Xiaohui and Lu, Jun and Kulbokas, E. J. and Golub, Todd R. and Mootha, Vamsi and Lindblad-Toh, Kerstin and Lander, Eric S. and Kellis, Manolis'
title: 'Systematic discovery of regulatory motifs in human promoters and 3, UTRs by comparison of several mammals'
---
authors: 'Zampelli, Stphane and Mann, Martin and Deville, Yves and Backofen, Rolf'
title: Techniques de Dcomposition pour l'Isomorphisme de Sous-Graphe
---
authors: 'Petersen, John E. and Englund, Goran'
doi: 10.1007/s00442-005-0062-z
title: 'Dimensional approaches to designing better experimental ecosystems: a practitioners guide with examples'
---
authors: 'Attanasio, Catia and David, Armelle and Neerman-Arbez, Marguerite'
doi: 10.1182/blood-2002-03-0853
title: Outcome of donor splice site mutations accounting for congenital afibrinogenemia reflects order of intron removal in the fibrinogen alpha gene (FGA)
---
authors: 'Amoutzias, Gregory D. and Robertson, David L. and Oliver, Stephen G. and Bornberg-Bauer, Erich'
doi: 10.1038/sj.embor.7400096
title: Convergent evolution of gene networks by single-gene duplications in higher eukaryotes
---
authors: 'Apaydin, Mehmet Serkan and Guestrin, Carlos E. and Varma, Chris and Brutlag, Douglas L. and Latombe, Jean-Claude'
title: Stochastic roadmap simulation for the study of ligand-protein interactions
---
authors: 'Bafna, Vineet and Tang, Haixu and Zhang, Shaojie'
title: Consensus Folding of Unaligned RNA Sequences Revisited
---
authors: 'Bartel, David P. and Chen, Chang-Zheng'
title: 'Micromanagers of gene expression: the potentially widespread influence of metazoan microRNAs'
---
authors: 'Pevzner, Pavel A. and Tang, Haixu'
title: Fragment assembly with double-barreled data
---
authors: 'Bussemaker, Harmen J. and Li, Hao and Siggia, Eric D.'
doi: 10.1073/pnas.180265397
title: 'Building a dictionary for genomes: Identification of presumptive regulatory sites by statistical analysis'
---
authors: 'Cannone, Jamie J and Subramanian, Sankar and Schnare, Murray N and Collett, James R and DSouza, Lisa M and Du, Yushi and Feng, Brian and Lin, Nan and Madabusi, Lakshmi V and Müller, Kirsten M and Pande, Nupur and Shang, Zhidi and Yu, Nan and Gutell, Robin R'
title: 'The Comparative RNA Web (CRW) Site: an online database of comparative sequence and structure information for ribosomal, intron, and other RNAs'
---
authors: 'Cline, M. S. and Shigeta, R. and Wheeler, R. L. and SIANI-ROSE, M. A. and Kulp, D. and Loraine, A. E.'
title: THE EFFECTS OF ALTERNATIVE SPLICING ON TRANSMEMBRANE PROTEINS IN THE MOUSE GENOME
---
authors: 'Dlakic, Mensur and Harrington, Rodney E.'
title: The effects of sequence context on DNA curvature
---
authors: 'Eddy, Sean R.'
title: Noncoding RNA genes
---
authors: 'Eddy, Sean R'
title: What is Bayesian statistics?
---
authors: 'Fekete, Martin and Hofacker, Ivo L. and Stadler, Peter F.'
title: Prediction of RNA Base Pairing Probabilities on Massively Parallel Computers
---
authors: 'Goto, S. and Bono, H. and Ogata, H. and Fujibuchi, W. and Nishioka, T. and Sato, K. and Kanehisa, M.'
title: ORGANIZING AND COMPUTING METABOLIC PATHWAY DATA IN TERMS OF BINARY RELATIONS
---
authors: 'Higo, Kenichi and Ugawa, Yoshihiro and Iwamoto, Masao and Korenaga, Tomoko'
title: 'Plant cis-acting regulatory DNA elements (PLACE) database: 1999'
---
authors: 'Khodabakhshi, Alireza Hadj and Manuch, JÁN and Rafiey, Arash and Gupta, Arvind'
doi: 10.1089/cmb.2008.0096
title: Stable Structure-Approximating Inverse Protein Folding in 2D Hydrophobic-Polar-Cysteine (HPC) Model
---
authors: 'Nebel, M. E.'
title: IDENTIFYING GOOD PREDICTIONS OF RNA SECONDARY STRUCTURE
---
authors: 'Pazos, Florencio and Valencia, Alfonso'
doi: 10.1002/prot.10074
title: In Silico Two-Hybrid System for the Selection of Physically Interacting Protein Pairs
---
title: ', Beyond the double helix'
---
authors: 'Pesole, Graziano and Liuni, Sabino and Grillo, Giorgio and Licciulli, Flavio and Mignone, Flavio and Gissi, Carmela and Saccone, Cecilia'
title: "UTRdb and UTRsite: specialized databases of sequences and functional elements of 5' and 3' untranslated regions of eukaryotic mRNAs. Update 2002"
---
authors: 'Reddy, Chandan K. and Chiang, HSIAO-DONG'
title: A Stability Boundary Based Method for Finding Saddle Points on Potential Energy Surfaces
---
authors: 'Rehmsmeier, Marc and Vingron, Martin'
doi: 10.1002/prot.XXXX
title: Phylogenetic Information Improves Homology Detection
---
authors: 'Spang, Rainer and Rehmsmeier, Marc and Stoye, Jens'
title: 'A Novel Approach to Remote Homology Detection: Jumping Alignments'
---
authors: 'Hofacker, Ivo L. and Fekete, Martin and Stadler, Peter F.'
title: Secondary Structure Prediction for Aligned RNA Sequences
---
authors: 'Vaughn, Matthew W. and Martienssen, Rob'
doi: 10.1126/science.1117805
title: "It's a Small RNA World, After All"
---
authors: 'Xu, Jinbo and Jiao, Feng and Berger, Bonnie'
doi: 10.1089/cmb.2007.R003
title: A Parameterized Algorithm for Protein Structure Alignment
---
authors: 'Fontana, Walter'
doi: 10.1002/bies.10190
title: Modelling 'evo-devo' with RNA
---
doi: 10.1186/1471-2148-8-89
title: ''
---
authors: 'Horton, Paul'
title: 'Tsukuba BB: A Branch and Bound Algorithm for Local Multiple Alignment of DNA and Protein Sequences'
---
authors: 'Jones, Richard B. and Carstens, Russ P. and Luo, Yongde and McKeehan, Wallace L.'
title: "5'- and 3'-terminal nucleotides in the FGFR2 ISAR splicing element core have overlapping roles in exon IIIb activation and exon IIIc repression"
---
authors: 'Kan, Z. and Castle, J. and Johnson, J. M. and Tsinoremas, N. F.'
title: DETECTION OF NOVEL SPLICE FORMS IN HUMAN AND MOUSE USING CROSS-SPECIES APPROACH
---
authors: 'Liu, Zesheng and Reches, Myriam and Groisman, Irina and Engelberg-Kulka, Hanna'
title: The nature of the minimal 'selenocysteine insertion sequence' (SECIS) in Escherichia coli
---
authors: 'Meshorer, Eran and Toiber, Debra and Zurel, Dror and Sahly, Iman and Dori, Amir and Cagnano, Emanuela and Schreiber, Letizia and Grisaru, Dan and Tronche, Francois and Soreq, Hermona'
doi: 10.1074/jbc.M402752200
title: Combinatorial Complexity of 5 Alternative Acetylcholinesterase Transcripts and Protein Products
---
authors: 'Randau, Lennart and Munch, Richard and Hohn, Michael J. and Jahn, Dieter and Soll, Dieter'
title: 'Nanoarchaeum equitans creates functional tRNAs from separate genes for their 5, - and 3, -halves'
---
authors: 'Bonneau, Richard and Tsai, Jerry and Ruczinski, Ingo and Chivian, Dylan and Rohl, Carol and Strauss, Charlie E. M. and Baker, David'
doi: 10.1002/prot.1170
title: 'Rosetta in CASP4: Progress in Ab Initio Protein Structure Prediction'
---
authors: 'Chen, Shuo and Zhang, Aixia and Blyn, Lawrence B. and Storz, Gisela'
doi: 10.1128/JB.186.20.6689-6697.2004
title: 'MicC, a Second Small-RNA Regulator of Omp Protein Expression in Escherichia coli'
---
authors: 'Frech, Kornelie and Danescu-Mayer, Joana and Werner, Thomas'
title: A Novel Method to Develop Highly Specific Models for Regulatory Units Detects a New LTR in GenBank which Contains a Functional Promoter
---
authors: 'Hinds, David A. and Levitt, Michael'
title: From Structure to Sequence and Back Again
---
authors: 'Ideker, Trey and Ozier, Owen and Schwikowski, Benno and Siegel, Andrew F.'
title: Discovering regulatory and signalling circuits in molecular interaction networks
---
authors: 'Koehl, Patrice and Levitt, Michael'
title: A brighter future for protein structure prediction
---
authors: 'Ouzounis, Christos A. and Coulson, Richard M. R. and Enright, Anton J. and Kunin, Victor and Pereira-Leal, José B.'
doi: 10.1038/nrg1113
title: CLASSIFICATION SCHEMES FOR PROTEIN STRUCTURE AND FUNCTION
---
authors: 'Pabo, Carl O. and Nekludova, Lena'
doi: 10.1006/jmbi.2000.3918
title: 'Geometric Analysis and Comparison of Protein-DNA Interfaces: Why is there no Simple Code for Recognition?'
---
authors: 'Romby, Pascale and Wagner, E. Gerhart H.'
doi: 10.1042/BC20070146
title: Exploring the complex world of RNA regulation
---
authors: 'Sagot, Marie-France and Myers, Eugene W.'
title: Identifying satellites and periodic repetitions in biological sequences
---
authors: 'Steipe, B.'
title: Evolutionary Approaches to Protein Engineering
---
authors: 'Voigt, Christopher A. and Benjamin Gordon, D. and Mayo, Stephen L.'
doi: 10.1006/jmbi.2000.3758
title: 'Trading Accuracy for Speed: A Quantitative Comparison of Search Algorithms in Protein Sequence Design'
---
authors: 'Wernisch, Lorenz and Hery, Stephanie and Wodak, Shoshana J.'
doi: 10.1006/jmbi.2000.3984
title: Automatic Protein Design with All Atom Force-fields by Exact and Heuristic Optimization
---
authors: 'Wolfe, Scot A. and Greisman, Harvey A. and Ramm, Elizabeth I. and Pabo, Carl O.'
title: 'Analysis of Zinc Fingers Optimized via Phage Display: Evaluating the Utility of a Recognition Code'
---
authors: 'Sebanz, Natalie and Knoblich, Gunther and Prinz, Wolfgang'
doi: 10.1016/S0010-02770300043-X
title: "Representing others' actions: just like one's own?"
---
authors: 'Glimcher, Paul W. and Rustichini, Aldo'
title: 'Neuroeconomics: The Consilience of Brain and Decision'
